packages/design-system-react-native/src/components/Icons/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import { useTailwind } from '@metamask/design-system-twrnc-preset';
+import type { Meta, StoryObj } from '@storybook/react-native';
+import React from 'react';
+import { ScrollView, View } from 'react-native';
+
+import Icon from './Icon';
+import {
+  DEFAULT_ICON_SIZE,
+  DEFAULT_ICON_NAME,
+  DEFAULT_ICON_COLOR,
+} from './Icon.constants';
+import type { IconProps } from './Icon.types';
+import { IconName, IconColor, IconSize } from './Icon.types';
+
+const meta: Meta<IconProps> = {
+  title: 'Components/Icon',
+  component: Icon,
+  argTypes: {
+    name: {
+      control: 'select',
+      options: IconName,
+    },
+    size: {
+      control: 'select',
+      options: IconSize,
+    },
+    color: {
+      control: 'select',
+      options: IconColor,
+    },
+    twClassName: {
+      control: 'text',
+    },
+  },
+};
+
+export default meta;
+
+const IconStory: React.FC<IconProps> = ({ color, ...props }) => {
+  const tw = useTailwind();
+  return (
+    <View
+      style={[
+        tw`${
+          color?.endsWith('-inverse')
+            ? 'bg-text-default'
+            : 'bg-background-default'
+        }`,
+      ]}
```

Comment:
May need to update to use individual primary default / primary inverse colors

Reply:
https://github.com/user-attachments/assets/59613f47-2556-4cfb-80b4-5a60c13bde84